marko/ext/gfm/renderer.py

```diff
@@ -104,3 +106,19 @@ def render_url(self, element):
     @render_url.dispatch(MarkdownRenderer)
     def render_url(self, element):
         return element.dest
+
+    @render_dispatch(HTMLRenderer)
+    def render_alert(self, element):
+        header = self.escape_html(element.alert_type)
+        children = self.render_children(element)
+        return f"<blockquote>\n<p>{header}</p>\n{children}</blockquote>\n"
```

Comment:
Maybe add a class name to the header p element so that users can style it with least effort